airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/NormalizationActivityImpl.java

```diff
@@ -176,6 +178,11 @@ public NormalizationSummary normalize(final JobRunConfig jobRunConfig,
 
   @Trace(operationName = ACTIVITY_TRACE_OPERATION_NAME)
   @Override
+  @Deprecated(forRemoval = true)
+  /**
+   * This activity is deprecated. It is using a big payload which is not needed, it has been replace
```

Comment:
what is big in `SyncInput`? Isn't catalog the big object?

Reply:
It is likely one of the biggest object. However the Sync Input also have unused object by the normalization such as State that can be big. The goal is to resolve the blob size issue face in https://airbytehq-team.slack.com/archives/C04NTJ82RC4/p1676046492611669?thread_ts=1676042334.857199&cid=C04NTJ82RC4. This is a short term fix which aims to unstuck a connection that is currently stuck.

Reply:
It also avoid having the catalog store in the SyncInput